runtime/src/in_mem_accounts_index.rs

```diff
@@ -481,8 +481,9 @@ impl<T: IndexValue> InMemAccountsIndex<T> {
     fn start_stop_flush(&self, stop: bool) {
         if stop {
             self.stop_flush.fetch_add(1, Ordering::Acquire);
```

Comment:
I know this isn't part of this PR, but I'm guessing this should probably be `Release` like the `else` case below. For one, we're not using the returned value from `fetch_add()` that we just read, which was a red flag. And if setting `stop_flush` to non-zero gates other logic, _writing_ this value is the important part for proper gating.

Reply:
done